### DISCUSSION TITLE
Enebling JIT prevents build.

### DISCUSSION THREAD

**Discussion by telisjok** [tailwindlabs/tailwindcss#4497] [Help]
Hi everyone,

I have an issue enabling JIT in my application. I followed tailwind's official docs for Enabling JIT mode by updating its dependencies (tailwindcss, postcss and autoprefixer) to the latest versions. The issue is that I am able to build the app locally including JIT (`mode: "jit"`) but when i'm trying to build it inside the container the building gets stuck here

`=> => #  DONE  Build complete. The dist directory is ready to be deployed.`
`=> => #  INFO  Check out deployment instructions at https://cli.vuejs.org/guide/deployment.html`

and doesn't proceed to deployment.

This only happens when I include `mode: "jit"` in `tailwind.config.js` file. If I remove it the problem goes away.

Any feedback would be highly appreciated.

**Comment by arunsingh28** [comment]
you can reinstall all package or delete package.json file then run  npm i or yarn add
